config/routes.rb

```diff
@@ -74,6 +74,7 @@
   end
 
   get "/downloads/:resource_id/file/:id", to: "downloads#show", as: :download
+  get "/manifests/:id/v3", to: "manifests#v3", as: :manifest_v3, defaults: { format: :json }
```

Comment:
Comment: We'll have to add this route to nginx_extra_config in princeton_ansible for CORS.

Reply:
https://github.com/pulibrary/princeton_ansible/issues/2551